cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,14 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: {{ .ImageRepository }}/busybox-{{ .Arch }}
```

Comment:
@ixdy We'd have to push an gcr.io/google_containers/busybox-${ARCH} image then (totally possible), don't know if that's feasible from a security standpoint (we'd have to keep it up to date)

cc @timstclair @thockin Please chime in with your thoughts here...